pixart.js

```diff
@@ -1 +1,41 @@
+var swatchArray = []
 
+for (i=0;i<8000;i++) {
+  var child = $("<div/>",{"class":"square"});
+  $("body").append(child);
+}
+
+for (i=0;i<3;i++){
+  var swatchChild = $("<div/>",{"class":"brush"});
+  $(".controls").append(swatchChild);
+}
+
+var setColor = function(evt){
+  evt.preventDefault();
+  var startColor = $(".brush").eq(0).css("background-color")
+  var colorSelect = $(":text").val();
+  $(".brush").eq(0).css("background-color", colorSelect);
+  var newColor = $(".brush").eq(0).css("background-color")
+  if(startColor != newColor){
+    swatchArray.unshift(colorSelect)
+    for(i=1;i<4;i++){
+      $(".brush").eq(i).css("background-color",swatchArray[i])
+    }
+  }
```

Comment:
wow this is intense! code comments would be helpful here to explain why you use both `.eq(0)` to set the style above and `.eq(i)` just above